draft-thomson-train-protocol.md

```diff
@@ -456,7 +456,25 @@ might benefit from understanding rate limits might consume rate limit signals
 when clients enable the feature. However, servers have to operate correctly even
 when clients act to protect the privacy of their service configuration.
 
+## Conditional Enablement
 
+As a signal that is passively observable means that its availability conveys
+information.  Making the availability of this capability conditional on client
+or server state leads to leaking information about that state.
+
```

Comment:
I have a hard time parsing the first sentence of that paragraph. Do you mean "The TRAIN is passively observable, and conveys information."? Also, not sure about "availabity. Do you mean "If the signal is only sent during specific client states, that provide information about those states" ?